imagepicker/src/main/java/com/esafirm/imagepicker/features/ImagePicker.java

```diff
@@ -132,6 +132,16 @@ public ImagePicker imageDirectory(String directory) {
         return this;
     }
 
+    public ImagePicker actionBarColor(int color) {
```

Comment:
I thinks it's better to add `@ColorInt` to this setter. 

Even better you can add `actionBarColorRes` with `@ColorRes` parameter 😸